app/serializers/application.js

```diff
@@ -1,27 +1,29 @@
 import RESTSerializer from '@ember-data/serializer/rest';
+import classic from 'ember-classic-decorator';
 import {camelize, decamelize, underscore} from '@ember/string';
 import {pluralize} from 'ember-inflector';
 
-export default RESTSerializer.extend({
+@classic
+export default class Application extends RESTSerializer {
     // hacky method for getting access to meta data for single-resource responses
     // https://github.com/emberjs/data/pull/4077#issuecomment-200780097
     // TODO: review once the record links and meta RFC lands
     // https://github.com/emberjs/rfcs/blob/master/text/0332-ember-data-record-links-and-meta.md
     extractMeta(store, typeClass) {
-        let meta = this._super(...arguments);
+        let meta = super.extractMeta(...arguments);
         typeClass.___meta = meta;
         return meta;
-    },
+    }
 
-    serialize(/*snapshot, options*/) {
-        let json = this._super(...arguments);
+    serialize/*snapshot, options*/() {
+        let json = super.serialize/*snapshot, options*/(...arguments);
```

Comment:
This looks a bit funky, I didn't realise you could use comment blocks quite like that 😄

Reply:
This looks wrong..